src/patternfly/components/Page/examples/page-component-drawer-example.hbs

```diff
@@ -0,0 +1,26 @@
+{{#> page page--nav-direction="vertical"}}
+  {{#> page-header}}
+    {{!-- Brand --}}
+    {{#> page-header-brand}}
+      {{#> page-header-brand-link}}
+        Logo
+      {{/page-header-brand-link}}
+    {{/page-header-brand}}
+    {{!-- Horizontal nav --}}
+    {{#> page-header-nav}}
+      pf-c-nav
+    {{/page-header-nav}}
+    {{!-- Toolbar --}}
+    {{#> page-header-tools}}
+      header-tools
+    {{/page-header-tools}}
+  {{/page-header}}
+  {{#> page-main}}
+    {{#> page-main-section page-main-section--modifier="pf-m-light"}}
+      page-main-section
+    {{/page-main-section}}
+  {{/page-main}}
+  {{#> page-drawer}}
+    page-drawer
```

Comment:
Should this example should the drawer expanded? Otherwise I think its a little unclear where it is

Reply:
good call, added